src/index.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, initial-scale=1, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <meta name="author" content="">
+  <meta name="description" content="Articles saver">
+  <title>Explorer</title>
+  <link rel="stylesheet" href="<%=htmlWebpackPlugin.files.chunks.main.css %>">
+  <link rel="icon" href="<%= require('./images/favicon.png')%>" type="image/png">
+</head>
+
+<body>
+<div class="root__section">
+  <header class="header">
+    <div class="menu">
+      <a href="./" class="menu__logo">Explorer</a>
+      <div class="menu__mobile" id="mobile-menu"></div>
+      <div class="menu__items-list">
+        <a href="./" class="menu__link menu__link_selected">Главная</a>
+        <div class="menu__items-single-item">
+            <span class="menu__link menu__link_logged">Юзер
+              <span class="menu__logout"></span>
+            </span>
+        </div>
+      </div>
+    </div>
+    <div class="header__title-wrapper">
+      <h1 class="header__title">Что в мире творится?</h1>
+      <p class="header__subtitle">
+        <b>Находите самые свежие статьи на любую тему
+          и сохраняйте в своём личном кабинете.</b>
+      </p>
+      <div class="header__form">
+        <input class="header__form-input" type="text" name="search" placeholder="Введите тему новости" required/>
+        <span class="button header__button">Искать</span>
```

Comment:
Tags should meet their semantic value. The button here should be represented by `button` or `input type="button"` tags.